### PR TITLE
smaller model

### PR DESCRIPTION
**Pull request by IgorPerikov**

### CODE REVIEW

File: src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt [hunk 1]
```diff
@@ -34,173 +21,126 @@ data class MatchData(
...
-        @JsonProperty("series_id") val seriesId: Int?,
-        @JsonProperty("series_type") val seriesType: Int?,
-        @JsonProperty("league") val league: League?,
-        @JsonProperty("radiant_team") val radiantTeam: RadiantTeam?,
-        @JsonProperty("dire_team") val direTeam: DireTeam?,
         @JsonProperty("players") val players: List<Player?>?,
         @JsonProperty("patch") val patch: Int?,
         @JsonProperty("region") val region: Int?,
-        @JsonProperty("all_word_counts") val allWordCounts: AllWordCounts?,
-        @JsonProperty("my_word_counts") val myWordCounts: MyWordCounts?,
```
Comment (F1nZeR) [inline src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt]: Chat might be interesting for researches.

File: src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt [hunk 2]
```diff
@@ -1,31 +1,18 @@
 package com.github.galleyltd.boost.opendota.dto
+
 import com.fasterxml.jackson.annotation.JsonProperty
 
 data class MatchData(
         @JsonProperty("match_id") val matchId: String?,
-        @JsonProperty("barracks_status_dire") val barracksStatusDire: Int?,
-        @JsonProperty("barracks_status_radiant") val barracksStatusRadiant: Int?,
-        @JsonProperty("chat") val chat: List<Chat?>?,
```
Comment (F1nZeR) [inline src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt]: Same.

File: src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt (hunk 1 above)
Comment (IgorPerikov) [reply src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt]: was defined as empty class:
```
class MyWordCounts
```

**Comment by F1nZeR** [comment]
LGTM 👍

File: src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt [hunk 3]
```diff
@@ -1,206 +1,147 @@
 package com.github.galleyltd.boost.opendota.dto
+
```
Comment (MysterionRise) [inline src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt]: ну пиздец

Comment (F1nZeR) [reply src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt]: поддерживаю

File: src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt [hunk 4]
```diff
@@ -241,103 +182,17 @@ data class GoldPerMin(
...
-
-class HeroHits
-
-class ItemUsage
-
-class FirstPurchaseTime
-
-class GoldReasons
-
-class AbilityUses
```
Comment (MysterionRise) [inline src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt]: not sure what's this, may be could be useful

File: src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt [hunk 5]
```diff
@@ -241,103 +182,17 @@ data class GoldPerMin(
...
     @JsonProperty("value") val value: Int?
 )
 
-class ItemWin
-
-class DamageInflictor
-
-class DamageTaken
-
-class ItemUses
```
Comment (MysterionRise) [inline src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt]: as well as this, but unsure why class is empty

File: src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt (hunk 4 above)
Comment (IgorPerikov) [reply src/main/kotlin/com/github/galleyltd/boost/opendota/dto/MatchData.kt]: How?

**Pull request merged**